$MirOS$

	This does limit names to the C locale, but skips error messages.

--- vfs/extfs/iso9660.in.orig	Fri Oct 29 09:14:16 2004
+++ vfs/extfs/iso9660.in	Thu Sep 28 20:38:17 2006
@@ -9,15 +9,7 @@
 # tested to comply with isoinfo 2.0's output
 
 test_iso () {
-    CHARSET=`echo "$save_ctype" | sed -n -e 's/.*\.\(.*\)"$/\1/p' | tr '[A-Z]' '[a-z]'`
-    if test -n "$CHARSET"; then
-	isoinfo -j $CHARSET 2>&1 | grep "Unknown charset" >/dev/null && CHARSET=
-    fi
-    if test -n "$CHARSET"; then
-	JOLIET_OPT="-j $CHARSET -J"
-    else
 	JOLIET_OPT="-J"
-    fi
     ISOINFO="isoinfo -R"
     isoinfo -d -i "$1" | grep "NO Joliet" > /dev/null || ISOINFO="$ISOINFO $JOLIET_OPT"
 }
@@ -65,7 +57,6 @@ mcisofs_copyout () {
 	$ISOINFO -i "$1" -x "/$2" > "$3" 
 }
 
-save_ctype=`locale | grep LC_CTYPE`
 LC_ALL=C
 
 cmd="$1"
